mkdocs-project-dir/docs/Account_Services.md

```diff
@@ -1,68 +1,137 @@
 ---
 title: Account Services
 categories:
- - User Guide
+- User Guide
 layout: docs
 ---
 
+<!--- ================================================================== --->
+<!---  @file   Account_Services.md                                       --->
+<!---  @brief                                                            --->
+<!---                                                                    --->
+<!---  @details                                                          --->
+<!---                                                                    --->
+<!---  @author(s)                                                        --->
+<!---  @contributor(s) e.kotsialos@ucl.ac.uk                             --->
+<!---  Copyright (c) 2020, UCL:ISD:RITS, RCAS                            --->
+<!---                                                                    --->
+<!--- ================================================================== --->
+
 # Account Services
 
 ## Cluster access application
 
-There is a single online form for applying to the majority of our clusters. Use of these services is subject to a common set of [terms and conditions](Terms_and_Conditions.md). All granted applications give you a Myriad account while access to other systems depends on what types of work you tell us you will be doing. 
+There is a single online form for applying to the majority of our clusters.
+Use of these services is subject to a common set of
+[terms and conditions](Terms_and_Conditions.md).
+All granted applications give you a Myriad account while access to other systems
+depends on what types of work you tell us you will be doing.
 
 ### Account sponsors
 
-If you are a student or postdoctoral researcher, your application must be approved by a permanent member of staff (normally your supervisor or PI). This will be automatically determined when you begin your application, and when you submit it, an email will be sent to this person asking them to approve the application before the account can be created.
+If you are a Student or Postdoctoral Researcher, your application must be
```

Comment:
Don't capitalise student, postdoctoral researcher, supervisor - used as descriptive terms, not titles. Don't think it is necessary to add PM (also don't know what that is...)